reana_commons/openapi_specifications/reana_server.json

```diff
@@ -742,6 +742,13 @@
             "name": "verbose",
             "required": false,
             "type": "boolean"
+          },
+          {
+            "description": "Size format, either 'k' (kilobytes) or 'b' (bytes).",
```

Comment:
Perhaps switch the order, first "b" then "k", to be sorted both alphabetically and size-wise

Reply:
... you can also harmonise help regarding "k", here it is "kilobytes" and in reana-client it is "kbytes" -- better to have the same term everywhere, just use "kilobytes"...